uzupelnianka.txt

```diff
@@ -1,7 +1,9 @@
-Wyjątki dzielimy na … które trzeba obsłużyć i … które można, ale nie trzeba obsłużyć.
+Wyjątki dzielimy na kontrolowane które trzeba obsłużyć i niekontrolowane które można, ale nie trzeba obsłużyć.
 
-Wyjątki … dziedziczą po klasie RuntimeException
+Wyjątki niekontrolowane dziedziczą po klasie RuntimeException
 
-Wyjątki rzucamy korzystając z instrukcji …, jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa …
+Wyjątki rzucamy korzystając z instrukcji throw new, jeśli rzucamy wyjątek kontrolowany
```

Comment:
> Wyjątki rzucamy korzystając z instrukcji throw new
Tak na prawdę, samo `throw`. `new` odnosi się do tworzenia obiektu (wyjątek to też obiekt). Można to rozbić na 2 linie.